content/05.discussion.md

```diff
@@ -1 +1,69 @@
-## Discussion
\ No newline at end of file
+## Discussion
+ 
+Scientific news coverage, unlike academic articles, is accountable to reflecting not only the current state of research, but also the diversity of its readership.
+News coverage is the conduit between the academic and public spheres, and consequently shapes the public's view of science and scientists.
+At a minimum, news coverage should match the current academic rates of regional and gender biases.
+To test whether this last point was true, we analyzed over 10,000 news articles published by Nature to identify quoted and cited persons.
+We then compared this to the authorship statistics from Nature's research articles and a subset of Springer's English language articles.
+
+We first looked at possible gender biases by examining the proportion of all identified quotes that were from a male speaker.
+As a comparator, we analyzed the proportion of male authors, which similarly is a measure of scientific participation.
+Using computational methods, we performed quote association and gender prediction.
+We observe a strong bias towards male participation across both authorship and quotes.
+We also identify a gender bias between first and last authors, previously shown in [CITE].
+Interestingly, we found that while the rate of male quotes and male authorship are decreasing over time, quotation rate is decreasing faster than authorship rates.
+We then tried to identify what drove the observed trend.
+First or last authorship changes, which we anticipated to affect gendered quotation rates, did not drive the downward trend.
+Despite first authors more commonly being female than last authors, first and last authors were quoted at comparable rates over time.
+Instead, column types were driving the trend.
+Our analysis shows that one article type, career-feature, has an equal number of quotes from both genders showing that gender parity is possible in science-related news coverage.
```

Comment:
I think this is really a key point here, as is the bit above that notes that the introduction of this article type appears to be one of the major drivers in increased parity. Right now this paragraph starts with just a restating of results, so this key point gets buried - especially if someone reading this says: "eh, I've heard this before" and stops reading before the last few sentences. I would restructure each of these results-related paragraphs to put the primary point that goes beyond strict interpretation of the data first, and then provide the support within the paragraph.

Reply:
Yeah, I think this suggestion made the discussion sound much stronger. Edited in https://github.com/greenelab/nature_news_manuscript/pull/1/commits/6e11c15a9953873cfee4ddff90e8a5e7cf7a9ebb